From dacf9830a6b40ab1c143e86938a8fc11d014cc16 Mon Sep 17 00:00:00 2001
From: Eugene Shalygin <eugene.shalygin@gmail.com>
Date: Wed, 27 Apr 2022 20:02:36 +0200
Subject: [PATCH 12/25] hwmon: (asus-ec-sensors) add doc entry for PRIME
 X470-PRO

Add PRIME X470-PRO to the list of supported boards.

Signed-off-by: Eugene Shalygin <eugene.shalygin@gmail.com>
Link: https://lore.kernel.org/r/20220427180237.1475954-1-eugene.shalygin@gmail.com
Signed-off-by: Guenter Roeck <linux@roeck-us.net>
---
 Documentation/hwmon/asus_ec_sensors.rst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/hwmon/asus_ec_sensors.rst b/Documentation/hwmon/asus_ec_sensors.rst
index 36ca531..1700fe6 100644
--- a/Documentation/hwmon/asus_ec_sensors.rst
+++ b/Documentation/hwmon/asus_ec_sensors.rst
@@ -4,6 +4,7 @@ Kernel driver asus_ec_sensors
 =================================
 
 Supported boards:
+ * PRIME X470-PRO,
  * PRIME X570-PRO,
  * Pro WS X570-ACE,
  * ProArt X570-CREATOR WIFI
-- 
2.37.0.3.g30cc8d0f14

